content/recipes/authoring/kata-snippets/full-tests.md

```diff
@@ -42,8 +42,7 @@ Some test suites require a reference solution to generate the expected value(s)
   - Input mutation by the user solution affecting the input passed to the reference solution
   - Incorrect implementation of the reference solution leading to rejection of valid user solutions
 - **The reference solution should not be revealed to the user.** When an assertion fails or the test suite crashes, some testing frameworks print fragments of source code which caused the failure to the console. It may happen that such printed failure messages or stack traces expose information about the solution which should not be revealed.
-- **The reference solution shouldn't be accessible to the user solution.** It should not be possible to implement the user solution as an alias or wrapper around the reference solution. The reference solution should be completely inaccessible outside the submit tests. In particular, it should not be a global and/or public function.
-Chek the [reference page for your language](/languages/) to see how to prevent this problem in your tests.
+- **The reference solution shouldn't be accessible to the user solution.** It should not be possible to call reference solution directly, or implement the user solution as an alias or wrapper around the reference solution. The reference solution should be completely inaccessible outside the submit tests. In particular, it should not be a global and/or public function. Chek the [reference page for your language](/languages/) to see how to prevent this problem in your tests.
```

Comment:
```suggestion
- **The reference solution shouldn't be accessible to the user solution.** It should not be possible to call the reference solution directly, or implement the user solution as an alias or wrapper around the reference solution. The reference solution should be completely inaccessible outside the submit tests. In particular, it should not be a global and/or public function. Check the [reference page for your language](/languages/) to see how to prevent this problem in your tests.
```